bloom-housing/bloom
# 1320/additional eligibility rules, listings management

# Pull Request Template

## Issue

Addresses #1320 

## Description

Adds the additional eligibility text box fields to the listings management prototype. All the fields already optionally showed.

## Type of change


## How Can This Be Tested/Reviewed?

Sign in as an admin to partners
Visit /listings/add
Check out the new additional eligibility section on this page
Submit a new listing as draft (add an existing property ID and a name and the four new fields)
Re-start the app
Check out the new listing on the public site and ensure the fields saved
In partners visit /listings/[id-of-listing-you-just-made]
Check out the new fields in that flavor of the form too

## Checklist:
